### DISCUSSION TITLE
Hosting on mogenius.com!

### DISCUSSION THREAD

**Discussion by Alii-isk** [pocketbase/pocketbase#909] [Q&A]
i just found out about this hosting platform that offers a decent free tier for starters without needing to a credit card.
so i was wondering if someone could make a guide on how to deploy pocketbase to **[mogenius.com](https://mogenius.com/)**? I would really appreciate it if someone could help me out with this!

**Comment by ganigeorgiev** [comment]
I haven't tested mogenius, but it seems like a tradition vps hosting, so basically you'll need to do 2 things:
1. Upload the PocketBase binary on the vps.
2. Create a `systemd`  service to keep the process running in case on server restart or similar.

You could find a detailed guide for DigitalOcean (which should be more-or-less the same) in https://github.com/pocketbase/pocketbase/discussions/512#discussioncomment-3668130.

**Reply by Alii-isk** [reply]
i just tested it out and there are using a docker
do i use same guide fly.io uses

**Reply by ganigeorgiev** [reply]
@Alii-isk Yes, you can use the same Dockerfile from the [fly.io guide](https://github.com/pocketbase/pocketbase/discussions/537).

If mogenius uses Docker, then you'll need to find how to register the persistent volume so that you can store your `pb_data` (_per their pricing page the free plan includes free "2 GB SSD persistent storage"_).
